### ISSUE TITLE
Error when using Class<T> type syntax

### ISSUE THREAD

**Issue by tibdex** [codemix/flow-runtime#73]
Hi, thanks for this amazing project. It's really powerful!
However, I'm having the following issue:

This is a:

 - [x] Bug Report
 - [ ] Feature Request
 - [x] Question
 - [ ] Other

Which concerns:

 - [x] flow-runtime
 - [ ] babel-plugin-flow-runtime
 - [ ] flow-runtime-validators
 - [ ] flow-runtime-mobx
 - [ ] flow-config-parser
 - [ ] The documentation website

---

### What is the current behaviour?

I have the following code:

```typescript
type HasKey = {
  getKey(): string
};

class Klazz {
  getKey() { return 'key'; }
}

const logKey = (C: Class<HasKey>) => {
 const instance = new C;
 console.log(instance.getKey()); 
}

logKey(Klazz);
```

When I run it in https://codemix.github.io/flow-runtime/#/try I get the following error:

> C must be a Class of HasKey
>
> Expected: Class<HasKey>
>
> Actual: Klazz

---

### What is the expected behaviour?

I would expect `flow-runtime` not to raise any error. Indeed, [this "Try Flow" playground ](https://flowtype.org/try/#0C4TwDgpgBAEghgZwNIRFAvFA3gKClAcwmBRAAoBKALigWACcBLAOwJwF8BuHHAYwBtECKEkEAvMdjyFipStij1iAV3rMoAcgDWqDZyjsOPXgHtmdKPxMFSGKGQDCNB4IQIAPPGSoAfBQw+UlCm5sBQLHRwzLzQmMwQAO5QDtzBZggm-BAAdFYEZBHAUTHZRCSolBT6Rjh5cqJwElVAA) does not report anything wrong and to me there is actually nothing wrong with the above code.

Is it a bug in `flow-runtime` or is it by design ? If it's by design, do you have a suggestion of how to do it differently ?

Thank you :)

---

### Which package versions are you using?

The one currently deployed at https://codemix.github.io/flow-runtime/#/try.

**Comment by phpnode** [comment]
it's a bug, I'll take a look

**Comment by phpnode** [comment]
released in v0.10.0

**Comment by tibdex** [comment]
Wow that was quick ! I downloaded _v0.10.0_ and it is indeed fix. However, it's still raising an error with https://codemix.github.io/flow-runtime/#/try. But I suppose it's only because the new docs haven't be deployed to _GitHub Pages_.
